internal/scaffold/crd_test.go

```diff
@@ -122,7 +123,54 @@ spec:
     storage: true
 `
 
-func TestCRDNonGoProject(t *testing.T) {
+func TestCRDNonGoProjectDefault(t *testing.T) {
+	s, buf := setupScaffoldAndWriter()
+	s.Fs = afero.NewMemMapFs()
+
+	r, err := NewResource(appApiVersion, appKind)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	crd := &CRD{Resource: r}
+	cfg := &input.Config{}
+	if err = s.Execute(cfg, crd); err != nil {
+		t.Fatalf("Failed to execute the scaffold: (%v)", err)
+	}
+
+	if crdNonGoDefaultExp != buf.String() {
+		diffs := diffutil.Diff(crdNonGoDefaultExp, buf.String())
+		t.Fatalf("Expected vs actual differs.\n%v", diffs)
+	}
+}
+
+// crdNonGoDefaultExp is the default non-go CRD. Non-go projects don't have the
+// luxury of kubebuilder annotations.
+const crdNonGoDefaultExp = `apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I beleve that it should be
```suggestion
const crdNonGoDefaultExp = `apiVersion: apiextensions.k8s.io/v1
```

Reply:
No, we are still using `v1beta1` because we want to continue to support pre-1.16 clusters. This PR does not change CRD generation to use `v1`. It simply adds valid structural schema to existing `v1beta1` CRDs.